### ISSUE TITLE
Ask for additional data

### ISSUE THREAD

**Issue by CChichoo** [chenjt66a/EB-LG-module#2]
Hello, thank you very much for the excellent project. 
I'm having some difficulty with successfully reproducing your accuracy.
if you don't mind, would you please provide me with a training loss graph or other more detailed hyperparameter settings?

**Comment by chenjt66a** [comment]
Thanks for your attention, you could run the pre-trained models we provide to reproduce some results. Actually, the records of training loss are not kept properly, and  the detailed hyperparameters we set are the same as the released codes, if i remember rightly.
